tests/test_transforms/test_inference/test_transform.py

```diff
@@ -17,6 +17,8 @@
 from etna.transforms import DensityOutliersTransform
 from etna.transforms import DeseasonalityTransform
```

Comment:
About failed inference tests:
- Check that two identical calls to encode models gives exactly the same result. May be we could set some random seed to make it more deterministic. If two subsequent calls give the same results it seems like the problem in procoessing different number of segments for some reason. Try to investigate it.
- If we can't fix the problem then we could increase the margins for checking equality. It seems like the results are very similar anyway.